jdaviz/configs/imviz/plugins/catalogs/catalogs.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+   <j-tray-plugin
+     description='Queries an area encompassed by the viewer using a specified catalog and marks all the objects found within the area.'
+    :link="'https://jdaviz.readthedocs.io/en/'+vdocs+'/'+config+'/plugins.html#catalog-search'"
```

Comment:
This link won't work until this PR is merged (I believe).

Reply:
until then [here are the rendered plugin docs](https://jdaviz--1455.org.readthedocs.build/en/1455/imviz/plugins.html#catalog-search)